ntbs-service/Models/Entities/User.cs

```diff
@@ -17,16 +17,6 @@ public class User
 
         public virtual ICollection<CaseManagerTbService> CaseManagerTbServices { get; set; }
 
-        [NotMapped] 
-        public string FullName => GivenName + " " + FamilyName;
-
-        [NotMapped]
-        public List<string> AdGroupNames { get; set; } = new List<string>();
-
-        internal void SetAdGroups(List<string> groups)
-        {
-            AdGroupNames = groups;
-            AdGroups = string.Join(",", groups);
-        }
+        [NotMapped] public string FullName => GivenName + " " + FamilyName;
```

Comment:
Separate lines?

Reply:
oh yeah, auto code clean-up got a bit busy :D but i guess i can just delete the notmapped altogether!